### PR TITLE
Added Rapier example

### PR DESCRIPTION
**Pull request by mrdoob**
**Description**

I've noticed the community is moving to Rapier so I wanted to have an example so we could compare.

It's nice that Rapier is actively maintained and the `RapierPhysics.js` is way less verbose.

### CODE REVIEW

File: examples/jsm/physics/RapierPhysics.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+	setInterval( step, 1000 / frameRate );
+
+	return {
+		addMesh: addMesh,
+		setMeshPosition: setMeshPosition
+	};
+
+}
+
+function compose( position, quaternion, array, index ) {
```
[the remark this answers is not in the document]
Comment (mrdoob) [reply examples/jsm/physics/RapierPhysics.js:207]: Oh yeah, when I did the original physics wrapper there was no import maps. Should be much cleaner now.

File: examples/jsm/physics/RapierPhysics.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+	const RAPIER = await import( path );
+	
+	await RAPIER.init();
+
+	const frameRate = 60;
+
+	// Docs: https://rapier.rs/docs/api/javascript/JavaScript3D/	
+
+	const gravity = { x: 0.0, y: - 9.81, z: 0.0 };
+    const world = new RAPIER.World( gravity );
```
[the remark this answers is not in the document]
Comment (mrdoob) [reply examples/jsm/physics/RapierPhysics.js:12]: A bit off how?

**Comment by mrdoob** [comment]
Increased the amount of elements:

https://raw.githack.com/mrdoob/three.js/dev/examples/physics_ammo_instancing.html
https://raw.githack.com/mrdoob/three.js/dev/examples/physics_rapier_instancing.html

File: examples/jsm/physics/RapierPhysics.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (LeviPesin) [reply examples/jsm/physics/RapierPhysics.js:12]: Screenshot_20230421-181023_Samsung Internet

[the remark this answers is not in the document]
Comment (Mugen87) [reply examples/jsm/physics/RapierPhysics.js:12]: Fixed 8ef744e70f80dfeba1a79cf2e86219f7915c4320.

**Pull request merged**